docs/topic_guides/evaluation/model.md

```diff
@@ -1,3 +1,33 @@
 # Model Evaluation
 
-This page is under construction - check back soon!
\ No newline at end of file
+The parameters of a trained Splink model build up a set of rules for scoring any pairwise records comparisons. Before scoring any pairs of records there are a number of ways to check whether your model will perform as you expect.
+
+## Look at the model parameters
+
+The final model is summarised in the [match weights chart](../../charts/match_weights_chart.ipynb) with each bar in the chart signifying the match weight (i.e. the amount of positive or negative evidence for a match) for each comparison level in your model.
+
+If, after some investigation, you still can't make sense of some of the match weights, take a look at the corresponding $m$ and $u$ values generated to see if they make logical sense. These can be produced with the [m u parameters chart](https://moj-analytical-services.github.io/splink/charts/m_u_parameters_chart.html).
+
+!!! info ""
+
+    Remember that $Match Weight = \log_2 \frac{m}{u}$
+
+
+## Look at the model training
+
+The process of training a model can offer some insight into its usefulness. The more stable a model is in the training process, the more reliable the outputs are.
+
+Stability of model training is seen clearly in the Expectation Maximaisation stage (for $m$ training):
+
+- Stability across EM training sessions can be seen through the [parameter estimates chart](https://moj-analytical-services.github.io/splink/charts/parameter_estimate_comparisons_chart.html)
+
+- Stability within each session is indicated by the speed of convergence of the algorithm. This is shown in the terminal output during training. In general, the fewer iterations required to converge the better.
+
+??? info "Example of convergence output"
+    ![](./image/convergence.png)
+
+## In summary
+
+Evaluating a trained model is not an exact science - there are no clear metrics to assess whether a model is good or bad at this stage. In most cases, applying human logic and heuristics is the best you can do to establish whether the model is sensible. Given the variety of potential use cases of Splink, there is no perfect, universal model, just models that can be tuned to produce useful outputs for a given application.
```

Comment:
I think there are metrics which we can use to assess good v bad to an extent (eg the params charts) but they aren't the end of the story or definitive
```suggestion
Evaluating a trained model is not an exact science - there are no metrics which can definitively say whether a model is good or bad at this stage. In most cases, applying human logic and heuristics is the best you can do to establish whether the model is sensible. Given the variety of potential use cases of Splink, there is no perfect, universal model, just models that can be tuned to produce useful outputs for a given application.
```